py-scripts/lf_ftp.py

```diff
@@ -296,17 +440,17 @@ def query_realclients(self):
                 raise ValueError("No Device is available to run the test hence aborting the test")
             logging.info("device got from webui are: %s", devices_list)
         else:
-            logging.info("AVAILABLE DEVICES TO RUN TEST : %s", self.user_list)
-            devices_list = input("Enter the desired resources to run the test:")
-            logging.info("devices list %s", devices_list)
+            devices_list = ""
         # print("devices list",devices_list)
         resource_eid_list = devices_list.split(',')
         resource_eid_list2 = [eid + ' ' for eid in resource_eid_list]
         resource_eid_list1 = [resource + '.' for resource in resource_eid_list]
         # print("resource eid list",resource_eid_list)
 
+        if devices_list == "" or devices_list == ",":
+            logger.warning("Can't run test on the selected devices")
+            exit(0)
```

Comment:
It is good practice to exit with a non-zero exit code when a program encounters an error. Please adjust this to something like `exit(1)`.